hooks/pre/10-reset_data.rb

```diff
@@ -2,58 +2,37 @@
 require 'English'
 
 def reset
-  if foreman_installed?
-
-    stop_services
-    start_services(['postgresql'])
-    reset_database
-    reset_candlepin
-    reset_pulp
-    reset_pulpcore if pulpcore_enabled?
-
-  else
-    Kafo::KafoConfigure.logger.warn 'Katello not installed yet, can not drop database!'
-  end
-end
-
-def foreman_installed?
-  `which foreman-rake > /dev/null 2>&1`
-  $CHILD_STATUS.success?
-end
-
-def load_foreman_config
+  stop_services
+  start_services(['postgresql']) if foreman_server? && local_postgresql?
```

Comment:
Why `foreman_server?` Isn't this sufficient?
```suggestion
  start_services(['postgresql']) if local_postgresql?
```